spring-boot-project/spring-boot-test-autoconfigure/src/main/resources/META-INF/spring/org.springframework.boot.test.autoconfigure.data.couchbase.AutoConfigureDataCouchbase.imports

```diff
@@ -1,5 +1,5 @@
 # AutoConfigureDataCouchbase auto-configuration imports
-
+org.springframework.boot.test.autoconfigure.service.connection.TestcontainersServiceConnectionAutoConfiguration
```

Comment:
Another idea I just had is having `META-INF/spring/org.springframework.boot.test.autoconfigure.data.cassandra.AutoConfigureDataCassandra.imports` and others in `spring-boot-testcontainers` module adding `org.springframework.boot.testcontainers.service.connection.ServiceConnectionAutoConfiguration`. I expect those will be merged when available and have the same result. Feedback is welcome